tests/app-template-minimal/app/app.js

```diff
@@ -0,0 +1,13 @@
+import Application from '@ember/application';
+import Resolver from 'ember-resolver';
+import config from './config/environment';
+
+const templates = import.meta.glob(`templates/**/*`, { eager: true });
```

Comment:
this isn't exactly what I'd want to do in a personal app, but if no components are in the templates folder, it seems ok -- probably wouldn't be compat with route splitting

Reply:
I think we can probably do a new feature of `@embroider/router` that lets you configure the route splitting directly in your `app/router.js`, and that code would directly do the corresponding `import.meta.glob`s. 

But also, even v2 addons can put things into `@embroider/virtual/compat-modules`, so maybe continuing to use that here makes sense.

Reply:
yea, that's a good point -- services are the obvious annoying-to-do-manually thing.
maybe this has gone too minimal!